src/G4VG.hh

```diff
@@ -38,8 +40,8 @@ struct Options
     //! Perform conversion checks
     bool compare_volumes{false};
 
-    //! TODO: allow client to use a different unit system (default: mm = 1)
-    static constexpr double scale = 1;
+    //! Value of 1mm in native unit system (0.1 for cm)
+    double scale = 1;
```

Comment:
Vecgeom "kind of" has a scale flag, but it's only used by VGDML. I think we should either move that flag into the vgdml loader or use it here.

P.S. @agheata @SeverinDiederichs the scale should *probably* be used in the "absolute" (non-relative) bump distances inside vecgeom, since otherwise loading and running the same problem at different length scales gives you different results.